tests/SshTest.php

```diff
@@ -83,6 +83,14 @@ public function it_can_enable_quiet_mode()
         $this->assertMatchesSnapshot($command);
     }
 
+    /** @test */
+    public function it_can_disable_password_authentication()
```

Comment:
Could you add a test for the `enable` variant as well. I think you'll find that the current code is broken.